sys/vm/vm_map.h

```diff
@@ -243,7 +243,9 @@ struct vm_map {
 #ifdef CHERI_CAPREVOKE
 	struct cv vm_cheri_revoke_cv;	/* (c) Cap. rev. is single file */
 	cheri_revoke_state_t vm_cheri_revoke_st;	/* Cap. rev. state */
-
+	cheri_revoke_state_t vm_cheri_async_revoke_st;
+	int vm_cheri_async_revoke_status;
+	const uint8_t * __capability vm_cheri_async_revoke_shadow;
 	/*
```

Comment:
Restore the blank line before the comment?

Reply:
In general we don't really adhere to that convention in struct definitions, I think? In this struct definition we already have several block comments without a leading newline.